src/model/types/account_id/eip155.rs

```diff
@@ -0,0 +1,155 @@
+use {
+    crate::model::types::erc55::erc_55_checksum_encode, once_cell::sync::Lazy, regex::Regex,
+    thiserror::Error,
+};
+
+// https://github.com/ChainAgnostic/namespaces/blob/main/eip155/caip10.md
+pub const NAMESPACE_EIP155: &str = "eip155";
+
+// https://github.com/ChainAgnostic/namespaces/blob/main/eip155/caip2.md#syntax
+static REFERENCE_PATTERN: Lazy<Regex> = Lazy::new(|| Regex::new(r"^\d+$").unwrap());
```

Comment:
same question about unwraps here

Reply:
Explained in d44a05581fe064d084da842dcf3715fd8ac855a3